src/Illuminate/Support/Arr.php

```diff
@@ -596,7 +596,7 @@ public static function sortRecursive($array)
 
     /**
      * Recursively sort an assoc array by keys.
-     * @param $array
+     * @param  array  $array
```

Comment:
Yeah this too, but I meant the empty line between the description and the params :)